scene/2d/cpu_particles_2d.cpp

```diff
@@ -166,10 +166,20 @@ void CPUParticles2D::_update_mesh_texture() {
 	vertices.push_back(-tex_size * 0.5 + Vector2(tex_size.x, tex_size.y));
 	vertices.push_back(-tex_size * 0.5 + Vector2(0, tex_size.y));
 	PoolVector<Vector2> uvs;
-	uvs.push_back(Vector2(0, 0));
-	uvs.push_back(Vector2(1, 0));
-	uvs.push_back(Vector2(1, 1));
-	uvs.push_back(Vector2(0, 1));
+	AtlasTexture *at = cast_to<AtlasTexture>(*texture);
```

Comment:
While that works since `CPUParticles2D` inherits `Object`, we usually use it as a static method with `Object::cast_to` spellt out fully.